### DISCUSSION TITLE
Password appears decoded in operator logs in error "Could not execute command"

### DISCUSSION THREAD

**Discussion by Mlk777** [OT-CONTAINER-KIT/redis-operator#992] [General]
**Describe your question**

Hi, we stumbled upon what we think is a problem when testing Redis resilience. 
The use case was to kill a pod to see what happens (follower for example) while monitoring the logs and check what was happening. 
The operator fails to do an action and we can see that the password is in clear is the msg "Could not execute command", cmd seems to be called in several places in the code but we didn't see the other info logs

**What version of redis-operator are you using?**

redis-operator version: v0.16.0

**Additional context**
 
![password operator](https://github.com/OT-CONTAINER-KIT/redis-operator/assets/22601983/aa8effb9-5197-42d7-ac08-c0cbe8ea200b)

Is it normal behavior or are we doing something wrong ? 

Thanks in advance

**Comment by drivebyer** [comment]
It's a bit tricky to determine. Just as you mentioned, you terminated a pod. Once a pod is terminated, the cluster might require some time to bounce back. Therefore, this error log isn't significant. The key is to check if the final state is satisfactory or not.

**Comment by Mlk777** [comment]
I'm closing the discussion, indeed on our side we had a sidecar doing operations that were colliding with the operator. It's now fixed, thank you
